states/CO/governor.yaml

```diff
@@ -0,0 +1,42 @@
+contact_form:
+  steps:
+    # Note that the form is inside of an iframe on this website: https://www.colorado.gov/governor/share-your-comments
+    - visit: "https://www.tfaforms.com/4679739?faIframeUniqueId=49cteb2vyx&hostURL=https%3A%2F%2Fwww.colorado.gov%2Fgovernor%2Fshare-your-comments"
+    # This is a radio button asking if the comment is related to proposed legislation, and checks the "no" radio button.
+    - javascript:
+        - value: "document.querySelector('#tfa_2746').checked = true;"
```

Comment:
Hm I think you can use `choose` directive here? From the contact-congress schema:

https://github.com/unitedstates/contact-congress/blob/master/documentation/schema.md#checkuncheckchoose